src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        return "Certain machine started its work";
```

Comment:
The message returned is not informative. It should clearly indicate that the Bulldozer is the machine that started working. Use 'Bulldozer' in the message to make it clear.